microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -490,4 +458,11 @@ Issued: 2024-03-20
 
 {product-title} release 4.14.17 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:1262[RHBA-2024:1262] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:1260[RHBA-2024:1260] advisory.
 
-For the `TopoLVM image`, see link:https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel9/63972de3adcb55263891b983?container-tabs=dockerfile[lvms4/topolvm-rhel9].
\ No newline at end of file
+For the `TopoLVM image`, see link:https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel9/63972de3adcb55263891b983?container-tabs=dockerfile[lvms4/topolvm-rhel9].
```

Comment:
Link specifies rhel9 but points to rhel8? This may be expected (not sure how the release timing works) but wanted to point out in case it was done in error. 
![image](https://github.com/openshift/openshift-docs/assets/122639474/41ea7f95-8dde-420e-a548-8f48b707eedc)

Reply:
Good callout. I need to look into that link as a whole.